packages/integrations/markdoc/README.md

```diff
@@ -458,6 +458,39 @@ To achieve a more Markdown-like experience, where HTML elements can be included
 > **Warning**
 > When `allowHTML` is enabled, HTML markup inside Markdoc documents will be rendered as actual HTML elements (including `<script>`), making attack vectors like XSS possible. Ensure that any HTML markup comes from trusted sources.
 
+### `allowIndentation`
+
+Enables indentation of Markdoc tags for better readability.
+
+The allowIndentation option is experimental and is not enabled by default. When the option is enabled, it is not possible to use indent-based code blocks.
```

Comment:
This could use an example for why you may want to configure this option. Is indentation usually not allowed, or is ignored? It isn't documented by markdoc.

Reply:
I found some documentation https://markdoc.dev/docs/faq#indentation - we could copy what's in there or just link to it?

Reply:
I found some documentation https://markdoc.dev/docs/faq#indentation - I've updated the README to reflect this